src/gui/Src/BasicView/Disassembly.cpp

```diff
@@ -116,6 +116,8 @@ void Disassembly::updateColors()
 void Disassembly::updateFonts()
 {
     setFont(ConfigFont("Disassembly"));
+    delete mFontMetrics;

```

Comment:
I have changed AbstractTableView a lot on my local branch. A new "Edit Columns” dialog is introduced to allow users to reorder/hide columns in a table view. That have to be pushed together with this pull request now. Just wait a moment please.